.github/workflows/ci.yml

```diff
@@ -53,14 +53,13 @@ jobs:
   static-stdlib:
     strategy:
       matrix:
-        os: [ubuntu-20.04]
-    runs-on: ${{ matrix.os }}
+        image: ['swift:5.7-focal', 'swift:5.8-focal']
+    runs-on: ubuntu-latest
+    container:
+      image: ${{ matrix.image }}
     steps:
-      - uses: swift-actions/setup-swift@v1
-        with:
-          swift-version: '5.8.0'
-      - name: Install dependencies
-        run: sudo apt-get install -y libcurl4-openssl-dev
```

Comment:
The advantage is that it avoids the `apt-get install` and allows easy extension to test it on 5.7 as well.

The downside is that the image doesn't come with `make`, so the `swift build` commands have to be inlined.

NB: this mechanism could also be used to extend the `ubuntu-tests` to cover other swift versions.